DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -19,43 +20,43 @@ final class SRSnapshotTests: SnapshotTestCase {
     private var shouldRecord = false
 
     func testBasicShapes() throws {
-        try takeSnapshotFor(.basicShapes, shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(Fixture.basicShapes, shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
```

Comment:
We need fully qualified names since the parameter is now `FixtureProtocol` instead of `Fixture`